src/coreclr/nativeaot/Runtime/ThunksMapping.cpp

```diff
@@ -25,7 +25,7 @@
 #elif TARGET_LOONGARCH64
 #define THUNK_SIZE  16
 #elif TARGET_RISCV64
-#define THUNK_SIZE  12
+#define THUNK_SIZE  24
```

Comment:
```suggestion
#define THUNK_SIZE  20
```

We removed one instruction.